### PR TITLE
Fix license headers

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -304,10 +305,14 @@
                                 <exclude>**/META-INF/persistence*.xsd</exclude>
                                 <exclude>**/pom.xml.versionsBackup</exclude>
                                 <exclude>ide-config/**</exclude>
+                                <exclude>**/*.mvel</exclude>
+                                <exclude>**/quarkus-extension.yaml</exclude>
```
Comment (ppalaga) [inline pom.xml]: I think we do not want the license headers in the quarkus-extension.yaml files. WDYT?

Comment (lburgazzoli) [reply pom.xml]: If it does not harm the quarkus processing I would say we should add to be consistent with other similar files (i.e. properties)

Comment (ppalaga) [reply pom.xml]: OK, removed the line.

**Comment by lburgazzoli** [comment]
ok to test

**Pull request merged**